system_tests/conftest.py

```diff
@@ -60,3 +63,80 @@ def inference_client(dar_url, credentials_source):
 def model_creator(dar_url, credentials_source):
     create_model = ModelCreator(dar_url, credentials_source)
     return create_model
+
+
+# Traceability report generation
+# The following hooks implement the traceability report generation.
+# This roughly works as follows:
+#
+# * In the runtest_setup hook, the issue ID is extracted from the closest
+#   `requirement` marker and stored on the test `item` object
+# * In the runtest_makereport hook, the issue ID is copied from the test `item`
+#   to the report object (basically, to the test result)
+# * Finally, the issue ID is consumed in the pytest_html_results_table_header and
+#   pytest_html_results_table_row hooks (provided by pytest-html plugin)
+#   and added to the HTML report
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers",
+        "requirements(issues=[issue1, issue2]): link to GitHub issue (traceability)",
```

Comment:
Does this mean only two issues can be linked to a test? Just curious.

Reply:
Good point. I added an ellipsis instead in 946c751 to show that multiple items can be linked.